src/pages/ProductDetail/ProductDetailArticle.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+function ProductDetailArticleList({ product }) {
+  return (
+    <article className="ProductDetailArticle">
+      <img
+        className="ProductDetailArticleImg"
+        alt="스킨"
+        src={product.subproductImg}
+      />
+      <div className="ProductDetailArticleContainer">
+        <div className="howUseUnderline">
+          <p className="subDetailFontColor">사용법</p>
+          <p className="howUseFontColor">{product.사용법}</p>
+        </div>
```

Comment:
```suggestion
        <div className="howUseUnderline">
          <p className="subDetailFontColor">사용법</p>
          <p className="howUseFontColor">{product.사용법}</p>
        </div>
```

* 반복되는 UI는 이 컴포넌트 전체가 아니라, 이 형태가 반복되고 있습니다
* 반복되는 형태를 상수데이터와 map으로 구현 해 주세요

Reply:
수정 완료 했습니다!